code/modules/surgery/tools.dm

```diff
@@ -8,6 +8,10 @@
 	w_class = WEIGHT_CLASS_TINY
 	origin_tech = "materials=1;biotech=1"
 
+/obj/item/retractor/suicide_act(mob/living/user)
+ 	user.visible_message("<span class='suicide'>[user] slams [src] into their eye! It looks like [user.p_theyre()] trying to commit suicide!</span>")

```

Comment:
ditto

Reply:
ditto to you too sir 🎩

Reply:
on a related note, this is why your build is failing, you've indented this with spaces and not tabs.

If you use a good text editor you can get it to show spaces and tabs differently

Reply:
( https://atom.io/ )

Reply:
ignore that comment, the person is clearly brain damaged

Reply:
fight me you kiwi fucker